matlab/schemas/+preprocess/Sync.m

```diff
@@ -30,6 +30,8 @@ function makeTuples(self, key)
                 packetLen = dat.analogPacketLen;
             end
             datT = pipetools.ts2sec(dat.ts, packetLen);
+            datT = datT-datT(1); % subtract offset
```

Comment:
Maybe this just demonstrates my ignorance of the sync again, but if you subtract `datT(1)` here but don't in `BehaviorSync`, how can the two timestamps ever match? They seem to refer to the same timestamps (both `dat.ts` loaded with a key from `Scan` with `preprocess.readHD5(key)`).